Vitis_Platform_Creation/Design_Tutorials/01-Edge-KV260/ref_files/step2_pfm/Makefile

```diff
@@ -17,7 +17,7 @@ PLATFORM_NAME=kv260_custom
 PLATFORM_NAME_FSBL=kv260_custom_fsbl
 XSA_NAME=kv260_hardware_platform
 VERSION=2022.1
-COMMON_IMAGE_PATH ?=${PLATFORM_REPO_PATHS}/sw/zynqmp/xilinx-zynqmp-common-v$(VERSION)/
+COMMON_IMAGE_ZYNQMP ?=${PLATFORM_REPO_PATHS}/sw/zynqmp/xilinx-zynqmp-common-v$(VERSION)/
```

Comment:
In our regression system, COMMON_IMAGE_ZYNQMP will also be set if the pre_exec is setup properly.
In users' system, their common image may not be stored under ${PLATFORM_REPO_PATHS}/sw/zynqmp/xilinx-zynqmp-common-v$(VERSION)/. So this line doesn't seem to be useful in either situation.